src/app/lib/analyticsUtils/eventListener/index.js

```diff
@@ -0,0 +1,45 @@
+import onClient from '../../utilities/onClient';
+import { getComponentsToTrack } from '..';
+import FrontPageAtiEventTracker from '../../../containers/ATIAnalytics/event/frontpage';
+
+const listener = (data, platform, statsDestination, service) => {
```

Comment:
Can `listener` take `FrontPageAtiEventTracker` as a prop? Would make it a little more reusable